src/App/Entity/Talk.php

```diff
@@ -77,57 +77,58 @@ public static function fromStandardTalk(
         DateTimeImmutable $time,
         Speaker $speaker,
         string $title,
-        string $abstract
+        string $abstract,
```

Comment:
Yes; it's difficult to reliably validate someone has entered "sufficient data"... do you require at least one character? two? ten? What characters are allowed? etc. etc.? It's better to simply say "it's required" IMO